### DISCUSSION TITLE
why use graphql-codegen and apollo-codegen at a time?

### DISCUSSION THREAD

**Discussion by crisfcodes** [saleor/saleor#9040] [General]
I've seen you use [graphql-codegen](https://www.graphql-code-generator.com/) in saleor-sdk and to a lesser extent on the saleor-dashboard... but I'm confused why you used in saleor-dashboard, graphql-codegen and apollo-codegen at the same time. Do you have some plan to migrate all to one tool or is a planned decision use both?.

**Comment by krzysztofwolski** [comment]
The plan is to migrate dashboard to graphql-codegen. Most likely it will be done during migration to Apollo Client v3.
